widgets/package.json

```diff
@@ -15,18 +28,37 @@
 		"@sveltejs/kit": "next",
 		"@tailwindcss/forms": "^0.3.3",
 		"@tailwindcss/line-clamp": "^0.2.1",
+		"@tsconfig/svelte": "^2.0.1",
 		"@types/dom-mediacapture-record": "^1.0.7",
 		"autoprefixer": "^10.2.6",
 		"postcss": "^8.3.1",
 		"postcss-load-config": "^3.0.1",
 		"prettier": "2.3.0",
 		"prettier-plugin-svelte": "2.3.0",
 		"svelte": "^3.34.0",
-		"svelte-check": "^2.0.0",
+		"svelte-check": "^2.2.3",
 		"svelte-preprocess": "^4.0.0",
+		"svelte2tsx": "^0.4.5",
 		"tailwindcss": "^2.1.4",
 		"tslib": "^2.0.0",
 		"typescript": "^4.0.0"
 	},
-	"type": "module"
+	"keywords": [
+		"hugging face",
+		"machine learning inference",
+		"svelte"
+	],
+	"exports": {
+		".": "./InferenceWidget/InferenceWidget.svelte",
+		"./WidgetHeader.svelte": "./InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte",
+		"./ModelPipelineIcon.svelte": "./ModelPipelineIcon/ModelPipelineIcon.svelte",
+		"./ModelPipelineTag.svelte": "./ModelPipelineTag/ModelPipelineTag.svelte",
+		"./inferenceSnippets": "./inferenceSnippets/index.js",
+		"./inferenceSnippets/inputs": "./inferenceSnippets/inputs.js",
+		"./interfaces/DefaultWidget": "./interfaces/DefaultWidget.js",
+		"./interfaces/Language": "./interfaces/Language.js",
+		"./interfaces/Libraries": "./interfaces/Libraries.js",
+		"./interfaces/Types": "./interfaces/Types.js"
```

Comment:
Can we have an `interfaces/index.js` that re-exports those, so that we can do `import { DefaultWidget, Language, ... } from "huggingface-widgets/interfaces"`?

Reply:
Does it make sense to do so? because, the use case has always been:
`import { x, y, z } from Interfaces or Language`,
not `import Interfaces`

Reply:
Yes, you're right actually! I think I'm feeling the effects of the jetlag x)